_ml-commons-plugin/api.md

```diff
@@ -729,8 +732,147 @@ GET /_plugins/_ml/stats
 }
 ```
 
+## Execute
+
+Some algorithms, such as [Localization]({{site.url}}{{site.baseurl}}/ml-commons-plugin/algorithms#localization), don't require trained models. You can run no-model-based algorithms using the `execute` API.
 
+```json
+POST _plugins/_ml/_execute/<algorithm_name>
+```
 
+### Example: Execute Localization 
 
+The following example uses localization to find subset level information for aggregate data (for example, aggregated over time) that demonstrates activity of interest such as spikes, drops, changes, and anomalies. 
```

Comment:
Change to match the earlier wording: "The following example uses the Localization algorithm to find subset-level information for aggregate data (for example, aggregated over time) that demonstrates the activity of interest, such as spikes, drops, changes, or anomalies."